app/controllers/suggestions_controller.rb

```diff
@@ -0,0 +1,33 @@
+require 'tunes_takeout_wrapper'
+
+class SuggestionsController < ApplicationController
+
+  def index
+    if params[:query]
+      @suggestion = TunesTakeoutWrapper.search(params[:query])
+      @restaurant = Restaurant.find(@suggestion.food_id)
+      @music = SpotifyItem.find_by(@suggestion.music_id, @suggestion.music_type)
+    end
+
+    @top_faves = TunesTakeoutWrapper.top_favorites
+
+    if current_user
+      @your_favorites = TunesTakeoutWrapper.user_favorites(current_user.uid)
```

Comment:
"your" can be unclear, i'd suggest using more explicit terminology for this variable (current_user_favs or something like that)